resolver/resolver.cc

```diff
@@ -2554,7 +2554,14 @@ void Resolver::sanityCheck(core::GlobalState &gs, vector<ast::ParsedFile> &trees
 ast::ParsedFilesOrCancelled Resolver::runIncremental(core::GlobalState &gs, vector<ast::ParsedFile> trees) {
     auto workers = WorkerPool::create(0, gs.tracer());
     trees = ResolveConstantsWalk::resolveConstants(gs, std::move(trees), *workers);
-    computeLinearization(gs);
+    // NOTE: Linearization does not need to be recomputed as we do not mutate mixins() during incremental resolve.
+    if (debug_mode) {
+        for (auto i = 1; i < gs.classAndModulesUsed(); i++) {
+            core::SymbolRef sym(gs, core::SymbolRef::Kind::ClassOrModule, i);
+            // If class is not marked as 'linearization computed', then we added a mixin to it since the last slow path.
+            ENFORCE_NO_TIMER(sym.data(gs)->isClassOrModuleLinearizationComputed(), sym.toString(gs));
+        }
+    }
```

Comment:
slight preference to use the `DEBUG_ONLY(...)` macro here.

Reply:
Done.